app/api/helpers/speaker.py

```diff
@@ -0,0 +1,24 @@
+from app.models.speakers_call import SpeakersCall
+from app.api.helpers.permission_manager import has_access
+from app.api.helpers.exceptions import ForbiddenException
+
+from datetime import datetime
+
+
+def can_edit_after_cfs_ends(event_id):
+    """
+    Method to check that user has permission to edit the speaker or session
+    after the CFS ends
+    """
+    speakers_call = SpeakersCall.query.filter_by(event_id=event_id).one()
+    if speakers_call:
+        speakers_call_tz = speakers_call.ends_at.tzinfo
+        if speakers_call.ends_at <= datetime.now().replace(tzinfo=speakers_call_tz) \
```

Comment:
Just use datetime.utcnow()

Reply:
@iamareebjamal - Won't work

`TypeError: can't compare offset-naive and offset-aware datetimes`

Reply:
We have to compare the current time (with the timezone of the speaker call) and the speaker call end date which already is timezone aware